.github/workflows/meshmap.yml

```diff
@@ -13,6 +13,7 @@ permissions:
 
 jobs:
   MeshMapScreenshot:
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository && always() }}
```

Comment:
Thanks. I'm also looking for this to be updated in the workflow template that is created when a user connects Meshery to their GitHub repo, @VaibhavMalik4187